RTClib.cpp

```diff
@@ -262,9 +274,12 @@ DateTime RTC_DS1307::now() {
   uint8_t m = bcd2bin(WIRE._I2C_READ());
   uint16_t y = bcd2bin(WIRE._I2C_READ()) + 2000;
   
-  return DateTime (y, m, d, hh, mm, ss);
+  DateTime dt = DateTime(y, m, d, hh, mm, ss);
+  if (!use_summertime_EU || !dt.summertime_EU(0)) return dt;
```

Comment:
`summertime_EU(0)` implies [Western European Time](https://en.wikipedia.org/wiki/Western_European_Time), i.e. UTC+00:00/UTC+01:00 (UK, Ireland and Portugal). Most of Europe uses Central European, Eastern European or Moscow time.